mobile/src/people/departments/departments-horizontal-scrollable-list.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { Component } from 'react';
+import { Animated, Easing, View, Text, ScrollView, Dimensions, TouchableOpacity, ScrollResponderEvent } from 'react-native';
+import { AppState } from '../../reducers/app.reducer';
+import { connect, Dispatch, MapStateToProps, MapDispatchToPropsFunction } from 'react-redux';
+import { EmployeeCardWithAvatar } from '../employee-card-with-avatar';
+import { DepartmentsTree } from './departments-tree';
+import { DepartmentsTreeNode } from './departments-tree-node';
+import { Employee } from '../../reducers/organization/employee.model';
+import { PeopleActions, requestEmployeesForDepartment, updateDepartmentIdsTree } from '../../reducers/people/people.action';
+
+interface DepartmentsHScrollableListProps {
+    departmentsTree: DepartmentsTree;
+    treeLevel?: number;
+    departmentsTreeNodes?: DepartmentsTreeNode[];
+    headDepartment: DepartmentsTreeNode;
+    topOffset?: number;
+}
+
+interface DepartmentsHScrollableListDispatchProps {
+    requestEmployeesForDepartment: (departmentId: string) => void;
+    updateDepartmentIdsTree: (index: number, departmentId: string) => void;
+}
+
+const mapDispatchToProps = (dispatch: Dispatch<PeopleActions>) => ({
+    requestEmployeesForDepartment: (departmentId: string) => { 
+        dispatch(requestEmployeesForDepartment(departmentId)); 
+    },
+    updateDepartmentIdsTree: (index: number, departmentId: string) => { 
+        dispatch(updateDepartmentIdsTree(index, departmentId)); 
+    },
+});
+
+export class DepartmentsHScrollableListImpl extends Component<DepartmentsHScrollableListProps & DepartmentsHScrollableListDispatchProps> {
+    private employeeCards: EmployeeCardWithAvatar[];
+    private animatedValue: Animated.Value;
+    private buttonText: Text;
+    private currentPage: number;
+
+    public componentDidMount() {
+        // this.animate.bind(this, Easing.bounce);
```

Comment:
Do you need this comments?

Reply:
Animation is still under development, so it will be changed soon.